lib/SurveyGizmo.js

```diff
@@ -10,16 +10,19 @@ class SurveyGizmo {
     this.getQuestion = this.getQuestion.bind(this);
     this.getQuestions = this.getQuestions.bind(this);
     this.getResponses = this.getResponses.bind(this);
-    this.getResponse = this.getResponse.bind(this);
-    this.getData = this.getData.bind(this);
+    this.getResponseData = this.getResponseData.bind(this);
   }
   /**
    * Makes AJAX request to get people responses.
+   * @param {number} responseId - single response ID. Pass this param if you want a single response.
    * @returns {object} JSON parsed API response.
    */
-  getResponses() {
+  getResponses(responseId) {
```

Comment:
I think it'd be better if this is separated by logic into two function: getResponses() and getResponse(id).